src/legacy/core_plugins/kibana/public/discover/components/field_chooser/discover_field_search.tsx

```diff
@@ -86,20 +225,25 @@ export function DiscoverFieldSearch({
           />
         </EuiFlexItem>
       </EuiFlexGroup>
-      <EuiFacetButton
-        aria-label={filterBtnAriaLabel}
-        data-test-subj="toggleFieldFilterButton"
-        className="dscToggleFieldFilterButton"
-        icon={<EuiIcon type="filter" />}
-        isSelected={filtersActive > 0}
-        quantity={filtersActive}
-        onClick={() => onShowFilter()}
-      >
-        <FormattedMessage
-          id="kbn.discover.fieldChooser.fieldFilterFacetButtonLabel"
-          defaultMessage="Fields filtered"
-        />
-      </EuiFacetButton>
+      <div className="lnsInnerIndexPatternDataPanel__filtersWrapper">
```

Comment:
I think you should copy this SCSS class to Discover's SCSS + rename it `dsc` prefixed

https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kibana/public/discover/components/field_chooser/_field_chooser.scss

Didn't look into it, but I guess there might be some unused CSS classes to cleanup inside